Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Xamarin.Forms.Controls.Issues.Shared.projitems

```diff
@@ -1161,7 +1161,7 @@
   <ItemGroup>
     <EmbeddedResource Include="$(MSBuildThisFileDirectory)A11yTabIndex.xaml">
       <SubType>Designer</SubType>
-      <Generator>MSBuild:Compile</Generator>
+      <Generator>MSBuild:UpdateDesignTimeXaml</Generator>
```

Comment:
Is this a necessary change?

Reply:
the IDE just makes it by default :-/ so I like to just get these kinds of these checked in otherwise it'll just keep making this change. This got defined wrong when it was initially checked in